src/components/hud-controller.js

```diff
@@ -19,10 +19,13 @@ AFRAME.registerComponent("hud-controller", {
     showTip: { type: "bool" }
   },
   init() {
+    this.onChildHovered = this.onChildHovered.bind(this);
     this.isYLocked = false;
     this.lockedHeadPositionY = 0;
     this.lookDir = new THREE.Vector3();
     this.lookEuler = new THREE.Euler();
+    this.store = window.APP.store;
```

Comment:
No need to store a reference here. I'd just use `window.APP.store` everywhere instead.

Reply:
maybe subjective, but i'd prefer to not be dereferencing the window object on every tick since that's kind of a deeper dependency on globals